src/plugins/management/public/management_sections_service.ts

```diff
@@ -94,8 +98,9 @@ export class ManagementSectionsService {
 
   start({ capabilities }: SectionsServiceStartDeps) {
     this.getAllSections().forEach((section) => {
-      if (capabilities.management.hasOwnProperty(section.id)) {
-        const sectionCapabilities = capabilities.management[section.id];
+      const capabilityId = MANAGEMENT_ID_TO_CAPABILITIES[section.id] || section.id;
```

Comment:
Is there an associated test to validate this is working correctly?

Reply:
+1

Reply:
@virajsanghvi unit test added